starter-code/jquery_ajax/main.js

```diff
@@ -12,7 +12,20 @@
 'use strict';
 (function() {
   var weatherUrl = "http://api.openweathermap.org/data/2.5/weather?q=";
-  var apiKey = "";
+  var userinput = "city";
```

Comment:
Here you should get the user input
```suggestion
  var userinput = $('#user').val();
```